.github/workflows/devops-test.yml

```diff
@@ -0,0 +1,263 @@
+name: DEVOPS-TEST
+
+on:
+  workflow_dispatch:
+    inputs:
+      JAVA_SDK_VERSION:
+        description: 'java sdk version'
+        required: true
+        default: '0.4.2'
+      OPENMLDB_SERVER_VERSION:
+        description: 'openmldb server version'
+        required: true
+        default: '0.4.2'
+      PYTHON_SDK_VERSION:
+        description: 'python sdk version'
+        required: true
+        default: '0.4.2'
+      BATCH_VERSION:
+        description: 'batch version'
+        required: true
+        default: '0.4.2-allinone'
+      DIFF_VERSIONS:
+        description: 'diff versions'
+        required: false
+        default: '0.4.0'
+      EXEC_TEST_TYPE:
+        description: 'Which tests need to be executed? The options are all, python, java, batch, cli, standalone-cli and apiserver'
+        required: true
+        default: 'all'
+
+env:
+  GIT_SUBMODULE_STRATEGY: recursive
+  HYBRIDSE_SOURCE:
+
+jobs:
+#  java-sdk-test-standalone-0:
```

Comment:
better no commented out jobs

Reply:
After the later stand-alone version test is completed, it will be opened